projects/demo/src/modules/experimental/card/examples/6/index.less

```diff
@@ -0,0 +1,27 @@
+@import 'taiga-ui-local';
+
+:host {
+    display: flex;
+    gap: 1rem;
+}
+
+.fade {
+    .scrollbar-hidden();
+    width: 100%;
+    height: 2rem;
+    white-space: nowrap;
+    overflow: auto;
+}
+
+.fade-vertical {
+    .scrollbar-hidden();
+    height: 6rem;
```

Comment:
It should be 5 and then you can remove height binding from the template

Reply:
fixed